docs/release/cut-sw-release.md

```diff
@@ -310,17 +310,40 @@ NON_UPCOMING_VERSIONS="<NON-UPCOMING VERSION(S)>"
 ```
 ```bash
 for ver in $NON_UPCOMING_VERSIONS; do
-    major_ver=`sed 's/.[0-9]*$//' <<< $ver`
+    major_ver="${ver%.*}"
     mv /tmp/$ver /usr/local/repo/tarball-install/$major_ver/
     rm -f /usr/local/repo/tarball-install/$major_ver/*latest*
 done
 /root/mk-sims.sh
 for ver in $NON_UPCOMING_VERSIONS; do
-    major_ver=`sed 's/.[0-9]*$//' <<< $ver`
+    major_ver="${ver%.*}"
     ls -l /usr/local/repo/tarball-install/$major_ver/*latest* # verify the symlinks are correct
 done
 ```
 
+Ask Tim Theisen, Brian Lin, or someone with privileges on the `opensciencegrid.org` repo servers to upload the tarballs with the following procedure:
+
+#### On a CS machine
```

Comment:
Does everyone on S&R have SSH access to `hcc-osg-software.unl.edu`? If they do not, what's the process for getting access?